addons/mail/static/src/models/thread/thread.js

```diff
@@ -1075,7 +1088,7 @@ function factory(dependencies) {
             }
             this.loadNewMessages();
             this.update({ isLoadingAttachments: true });
```

Comment:
This (and the one below) should probably be removed from here, as it is now done in the new method.